### ISSUE TITLE
TextToSpeech Throwing Error INDX( t�

### ISSUE THREAD

**Issue by dgrimey** [joomla/joomla-cms#38982]
### Steps to reproduce the issue
1. Not sure how to reproduce. Working on new site and it was fine on Friday; however, when I got started on it on Monday the error show up in the top of each page. I have an older version of the site installed in the same XXAMP install. They are both Joomla 4.2.3 and have almost all the same plugins. 

### Expected result
I have Accessibility plug "System - Additional Accessibility Features" turned off. 

### Device and system info

* What version of Joomla were you using? 4.2.3 - Using Gantry 5 - 

* What device were you using? (eg, phone, laptop) Desktop

* What web browser were you using? (eg, Chrome, Safari) Firefox, Chrome, and Edge

* If you were using assistive technology, please let us know the device or software you were using (eg VoiceOver, JAWS, sip-and-puff device, etc)
NONE

### Additional comments

If you know which [WCAG](https://www.w3.org/TR/WCAG/) or [ATAG](https://www.w3.org/TR/ATAG20/) criteria this violates, please indicate it here:
![joomla-4-error-2](https://user-images.githubusercontent.com/22226227/196240982-35d40cc6-21e1-4d25-9b17-cb744443f034.JPG)
![joomla-4-error-4](https://user-images.githubusercontent.com/22226227/196241038-6638965b-14fa-4474-bce4-48a0aba44225.JPG)
![joomla-4-error-2](https://user-images.githubusercontent.com/22226227/196241039-827b6fcc-24af-4800-8063-e2cc60d2f8ba.JPG)

I don't know.

**Comment by dgrimey** [comment]
Restored site from a back up and I am not seeing the issue.
